src/security/CookieSecurity.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var _ = require('lodash');
+var Security = require('./security');
+
+/*
+ * Accepts either a cookie or lastResponseHeaders
+ */
+class CookieSecurity extends Security {
+  constructor(cookie, options) {
+    super(options);
+
+    cookie = _.get(cookie, 'set-cookie', cookie)
+    var cookies = _.map(_.isArray(cookie) ? cookie : [cookie], function (c) {
```

Comment:
Something like this would be better I think. Less dependency on lodash
```
this.cookie = (Array.isArray(cookie) ? cookie : [cookie])
  .map(c => c.split(';')[0])
  .join('; ');
```